app/backend/config.py

```diff
@@ -5,9 +5,11 @@
 CONFIG_CHAT_VISION_APPROACH = "chat_vision_approach"
 CONFIG_CHAT_APPROACH = "chat_approach"
 CONFIG_BLOB_CONTAINER_CLIENT = "blob_container_client"
+CONFIG_BLOB_CONTAINER_CLIENTS = "blob_container_clients"
 CONFIG_AUTH_CLIENT = "auth_client"
 CONFIG_GPT4V_DEPLOYED = "gpt4v_deployed"
 CONFIG_SEMANTIC_RANKER_DEPLOYED = "semantic_ranker_deployed"
 CONFIG_VECTOR_SEARCH_ENABLED = "vector_search_enabled"
 CONFIG_SEARCH_CLIENT = "search_client"
```

Comment:
Hetzelfde geldt voor deze?

Reply:
When removing the singe blob_container_client and search_client configurations, some errors popped up. Will perform one final check based on the DnB setup and then merge

Reply:
Removed the single clients after further investigation.